src/ui/toolbar/MainToolBar.cc

```diff
@@ -68,8 +70,12 @@ MainToolBar::MainToolBar()
     }
     // Get screen DPI to manage font sizes on different platforms
     QScreen *srn = QGuiApplication::screens().at(0); // TODO: Find current monitor as opposed to picking first one
-    _dotsPerInch = (qreal)srn->logicalDotsPerInch(); // Font point sizes are based on Mac 72dpi
-
+    if(srn && srn->logicalDotsPerInch() > 50.0) {
+        _dotsPerInch = (qreal)srn->logicalDotsPerInch(); // Font point sizes are based on Mac 72dpi
+    } else {
+        qWarning() << "System not reporting logical DPI.";
```

Comment:
Might be worth stating that a specific default would be used. If we're spitting out warnings, we should give users a way to understand what was done anyways and possibly what they should do to fix this. If I saw this warning my first thought would be, okay, what kind of screen resolution is it going to use if it can't find a valid value?